lib/perl/Genome/Model/Command/Define/ImportedReferenceSequence.pm

```diff
@@ -359,29 +351,17 @@ sub _create_build {
         return;
     }
 
-    my @dispatch_parameters;
-    if(defined($self->server_dispatch)) {
-        push @dispatch_parameters,
-            server_dispatch => $self->server_dispatch;
-    }
-
-    if(defined($self->job_dispatch)) {
-        push @dispatch_parameters,
-            job_dispatch => $self->job_dispatch;
-    }
-
     $self->status_message('Starting build.');
-    if($build->start(@dispatch_parameters)) {
-        if($self->server_dispatch eq 'inline') {
-            $self->status_message('Reference imported. ID: <' . $build->id . '>, data_directory: <' . $build->data_directory . '>.');
-        } else {
-            $self->status_message('Started build.');
-        }
+    if($build->start) {
+        UR::Context->commit or die "Failed to commit.";
+        $self->status_message('Started build.');
     } else {
         $self->error_message("Failed to start build " . $build->build_id . " for model " . $model->genome_model_id . ".");
         return;
     }
 
+    $build->wait;
```

Comment:
Perhaps waiting should be optional if the user doesn't want to wait?  (That was the effect of overriding the server dispatch previously.)